jeff65/gold/ast.py

```diff
@@ -387,7 +391,28 @@ def __init__(self, position, text):
 
 
 class IdentifierNode(TermNode):
-    pass
+    def __init__(self, position, text):
+        super().__init__(position, text)
+        self.member_index = None
```

Comment:
TODO: maybe i should just append the `BracketsNode` to the end of `self.children`?

i do feel like it needs to be a child of `IdentifierNode`. In `x[0] = 5`, the entire `x[0]` part should be considered one big identifier.